.gitattributes

```diff
@@ -3,4 +3,3 @@
 #
 # These are explicitly windows files and should use crlf
 *.bat           text eol=crlf
-
```

Comment:
This is a rebase issue. Please, perform rebase properly and get rid of any merge commits.